Modules/Tags.ConsumerApi/Controllers/TagsController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Backbone.BuildingBlocks.API.Mvc;
+using Backbone.Modules.Tags.Application.Tags.Queries.ListTags;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backbone.Modules.Tags.ConsumerApi.Controllers;
+
+[Route("api/v1/[controller]")]
+public class TagsController : ApiControllerBase
```

Comment:
You should probably add the `[Authorize("OpenIddict.Validation.AspNetCore")]` attribute to the controller, and the `AllowAnonymous` attribute to the controller method. This makes sure that when adding a new route we don't accidentally make it public.